workflows/alevin-quant/run-alevin.nf

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
+params.index_dir = 'salmon_index'
+params.index_name = 'cdna_k31'
+params.annotation_dir = 'annotation'
+params.t2g = 'Homo_sapiens.ensembl.100.tx2gene.tsv'
+params.mitolist = 'Homo_sapiens.ensembl.100.mitogenes.txt'
+params.sample_dir = 's3://ccdl-scpca-data/raw/green_adam'
+params.sample_id = ['834', '905_3']
+params.outdir = 's3://nextflow-ccdl-results/scpca/alevin-quant'
+
+// build full paths
+params.index_path = "${params.ref_dir}/${params.index_dir}/${params.index_name}"
+params.t2g_path = "${params.ref_dir}/${params.annotation_dir}/${params.t2g}"
+params.mito_path = "${params.ref_dir}/${params.annotation_dir}/${params.mitolist}"
+
+process alevin{
+  container 'quay.io/biocontainers/salmon:1.3.0--hf69c8f4_0'
+  cpus 8
+  // try dynamic memory (28.GB so 2x will fit in r4.2xlarge)
+  memory { 28.GB * task.attempt}
+  errorStrategy { task.exitStatus in 137..140 ? 'retry' : 'terminate' }
+  maxRetries 1
+  tag "${id}-${index}"
+  publishDir "${params.outdir}"
+  input:
+    tuple val(id), path(read1), path(read2)
+    path index
+    path tx2gene
+  output:
+    path "${id}-${index}"
```

Comment:
It is in this case, but does not need to be. I am actually unsure whether it can be replaced with a separate variable. They do not really need to stay in sync, as `tag` is only used to ease tracking of individual processes.